jujugui/static/gui/src/app/components/user-profile/test-user-profile.js

```diff
@@ -63,6 +63,7 @@ describe('UserProfile', () => {
     var getAgreements = sinon.stub();
     var staticURL = 'test-url';
     var user = users.charmstore;
+    window.flags = {blues: true};
```

Comment:
This flag needs to be unset at the end of this test even if the test fails halfway through. This we'll typically do in the afterEach() so that it doesn't cascade into other tests.

But maybe instead we should be passing in the flags into the component when it's necessary so that we don't have to use the globals at all.